### PR TITLE
Update cats to 2.2.0, slick-cats to 0.10.3

### PR DESCRIPTION
**Pull request by karolchmist**
A new version of cats is available, with faster compilation times.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +51,7 @@ Artifacts are publicly available on Maven Central starting from version *0.6*.
 Some or all of the following imports may be needed:
 ```scala
 import cats._
-import cats.implicits._
+import cats.syntax.all._
```
Comment (karolchmist) [inline README.md:54]: Cats 2.2.0 has a new import for faster compilation times. It requires to remplace `cats.implicits._` with `cats.syntax.all._`:
https://github.com/typelevel/cats/releases/tag/v2.2.0

File: slick-cats/src/test/scala/com/rms/miu/slickcats/DBIOInstancesTest.scala [hunk 2]
```diff
@@ -9,7 +9,7 @@
 package com.rms.miu.slickcats
 
 import cats.data.EitherT
-import cats.implicits._
+import cats.syntax.all._
```
Comment (Tvaroh) [inline slick-cats/src/test/scala/com/rms/miu/slickcats/DBIOInstancesTest.scala:12]: Could you make it in alphabetic order, please.

**Review by Tvaroh — APPROVED** [review]
Thanks!

File: slick-cats/src/test/scala/com/rms/miu/slickcats/DBIOInstancesTest.scala (hunk 2 above)
Comment (karolchmist) [reply slick-cats/src/test/scala/com/rms/miu/slickcats/DBIOInstancesTest.scala]: Done

**Review by 23will — APPROVED** [review]

**Pull request merged**